databases/week3/hyf_lesson3.sql

```diff
@@ -0,0 +1,196 @@
+CREATE DATABASE hyf_lesson3;
+USE hyf_lesson3;
+SET NAMES 'utf8mb4';
+CREATE TABLE `meal`(
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `title` VARCHAR(255) NOT NULL,
+    `description` TEXT NULL DEFAULT NULL,
+    `location` VARCHAR(255) NOT NULL,
+    `when` DATETIME NOT NULL,
+    `max_reservations ` INT(10) UNSIGNED NOT NULL,
+    `price` DECIMAL NOT NULL,
```

Comment:
don't forget to specify how many digits you want in total and how many decimal places.